ddtrace/llmobs/_utils.py

```diff
@@ -17,6 +17,33 @@
 log = get_logger(__name__)
 
 
+def validate_prompt(prompt: dict) -> dict:
+    validated_prompt = {}
+    if not isinstance(prompt, dict):
+        raise TypeError("Prompt must be a dictionary")
+    if prompt.get("variables"):
+        variables = prompt["variables"]
+        if not isinstance(variables, dict):
+            raise TypeError("Prompt variables must be a dictionary.")
+        for key, value in variables.items():
+            if not isinstance(key, str) or not isinstance(value, str):
+                raise TypeError("Prompt variable keys and values must be strings.")
+        validated_prompt["variables"] = prompt["variables"]
+    if prompt.get("template"):
+        if not isinstance(prompt["template"], str):
+            raise TypeError("Prompt template must be a string")
+        validated_prompt["template"] = prompt["template"]
+    if prompt.get("version"):
+        if not isinstance(prompt["version"], str):
+            raise TypeError("Prompt version must be a string.")
```

Comment:
Does version really need to be a string? Int is not accepted?

Reply:
Our backend only accepts strings, I think to keep the data model consistent with the ingest API we should only accept a strings in the SDK as well